### PR TITLE
Feature/write readme

### PR DESCRIPTION
**Pull request by Atos20**
#### What’s this PR do?  
- Modifies the readme.md to give the context of the work done over the course of the time working of the project
- Adds gifs to the read me
- Adds contributors
- Adds set-up instructions
- Adds, challenges, wins, technologies used, and project directions

#### Where should the reviewer start?  
 `readme.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,103 @@
...
-## Deploying to GitHub Pages
-
-_If you are finished with the functionality and testing of your project_, then you can consider deploying your project to the web! This way anyone can play it without cloning down your repo.
-
-[GitHub Pages](https://pages.github.com/) is a great way to deploy your project to the web. Don't worry about this until your project is free of bugs and well tested!
-
-If you _are_ done, you can follow [this procedure](./gh-pages-procedure.md) to get your project live on GitHub Pages.
+[![Contributors][contributors-shield]](https://github.com/Atos20/what-s-cooking-refactor/graphs/contributors)
+[![Forks][forks-shield]](https://github.com/Atos20/what-s-cooking-refactor/network/members)
+[![Issues][issues-shield]](https://github.com/Atos20/what-s-cooking-refactor/issues)
```
Comment (Atos20) [inline README.md:3]: We don't need to have this part but added just in case you all think it is a good idea.

File: README.md [hunk 2]
```diff
@@ -1,105 +1,103 @@
...
+
+## Foodamental _*group project*_
+
+## Contributors
+  - [Kyle Jones](https://github.com/KJJones13)
+  - [Blake Donn](https://github.com/BlakeDonn)
```
Comment (Atos20) [inline README.md:9]: just make sure your links are working. They are working on my end but just to be sure

File: README.md [hunk 3]
```diff
@@ -1,105 +1,103 @@
...
+[forks-url]: https://github.com/jordy1611/whats-cookin-JS-JS-KS/network/members
+[stars-shield]: https://img.shields.io/github/stars/jordy1611/whats-cookin-JS-JS-KS.svg?style=flat-square
+[stars-url]: https://github.com/jordy1611/whats-cookin-JS-JS-KS/stargazers
+[issues-shield]: https://img.shields.io/github/issues/jordy1611/whats-cookin-JS-JS-KS.svg?style=flat-square
+[issues-url]: https://github.com/jordy1611/whats-cookin-JS-JS-KS/issues
+[license-shield]: https://img.shields.io/github/license/jordy1611/whats-cookin-JS-JS-KS.svg?style=flat-square
+[license-url]: https://github.com/jordy1611/whats-cookin-JS-JS-KS/blob/master/LICENSE.txt
+[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat-square&logo=linkedin&colorB=555
+[linkedin-url]: https://linkedin.com/in/othneildrew
+[product-screenshot]: images/screenshot.png
```
Comment (Atos20) [inline README.md:103]: We are not using all these links so we can do a little bit of clean up

File: README.md [hunk 4]
```diff
@@ -1,105 +1,103 @@
...
+  * SCSS
+  * Dependencies
+
+## Challenges
+
+    - API's
+    - manipulation of nested data types
+    - use of chai-spies to spy on DOM related methods
+
+## Wins
```
Comment (Atos20) [inline README.md:45]: Added some of  my winds but please feel free to add some of yours, we could potentially change the title to `team wins`

File: README.md [hunk 5]
```diff
@@ -1,105 +1,103 @@
...
+
+#### Technologies used:
+  * JavaScript ES6
+  * Terminal
+  * GitHub
+  * HTML
+  * SCSS
+  * Dependencies
+
+## Challenges
```
Comment (Atos20) [inline README.md:39]: I think we can change this to tea challenges or something like that, what do you all think?

**Review by BlakeDonn — APPROVED** [review]
Looks good to me, good thorough instructions for installation

**Pull request merged**